File: Front-End/src/app/Modelos/auto.ts
export class Auto {
    public id : string;
    public matricula : string;
    public marca : string;
    public modelo : string;
    public autosAutomivilistas: Array<any>;

    public constructor( data: any = {}) {
        this.id = data.id || '' ;
        this.matricula = data.matricula || '' ;
        this.marca = data.marca || '' ;
        this.modelo = data.modelo || '' ;
        this.autosAutomivilistas = data.autosAutomivilistas || null;
    }   
}

File: Back-End/Entidades/Auto.cs
namespace Entidades
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.Text;

    public class Auto
    {
        public Guid Id { get; set; }
        [StringLength(30)]
        public string Matricula { get; set; }
        [StringLength(80)]
        public string Marca { get; set; }
        [StringLength(80)]
        public string Modelo { get; set; }

        public List<AutoAutomivilista> AutosAutomivilistas { get; set; }

    }
}
